### PR TITLE
AT-1221 - Add actionbar modifier

### PR DESCRIPTION
**Pull request by johannacarlberg**
Added a modifier to nav-toolbar called compact. With this modifier there will be no top-padding on the action-buttons on large screen, it will remain 16px on any screen smaller than large

### CODE REVIEW

File: less/navbar.less [hunk 1]
```diff
@@ -1191,6 +1191,14 @@
   }
 }
 
+.nav-toolbar--compact {
+  padding-top: 16px;
```
Comment (andreapiras) [inline less/navbar.less]: Can we use ```@spacer``` here?

**Review by andreapiras — APPROVED** [review]

File: less/navbar.less [hunk 2]
```diff
@@ -1191,6 +1191,14 @@
   }
 }
 
+.nav-toolbar--compact {
+  padding-top: (@spacer * 2);
```
Comment (nabaroa) [inline less/navbar.less]: Is this line necessary? the .nav-toolbar has already that padding.

Comment (johannacarlberg) [reply less/navbar.less]: you are right, not needed, will update!

**Review by nabaroa — APPROVED** [review]
🚀

**Pull request merged**